src/plugins/embeddable/public/lib/editable_embeddable/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Any reason this file needs to go in the `embeddable` plugin vs the `presentation_publishing` package?

Reply:
I tried to put it there originally, but I remember there was something I needed from the `embeddable` plugin that I couldn't import and it was causing issues.... I think `EmbeddableAppContext`?